xml/Microsoft.VisualBasic/Interaction.xml

```diff
@@ -1064,7 +1064,7 @@ drawObj.SaveAs("C:\Drawings\sample.drw")
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `IIf` function provides a counterpart for the ternary [Conditional Operator: ? :](https://msdn.microsoft.com/library/88643ee8-7100-4f86-880a-705ec22b6271) in Visual C++.  
+ The `IIf` function provides a counterpart for the ternary [Conditional Operator: ? :](/cpp/cpp/conditional-operator-q?view=msvc-160) in Visual C++.  
```

Comment:
Not sure what are the rules on this at the moment, but I usually remove view params to always go to the latest.